### PR TITLE
Feature/cor 1498 vaccinations toggle button removal cherrypick

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard27**
## Summary

- Removed toggle button, keys, vr connections at  [vaccine-coverage-choropleth.tsx](https://github.com/minvws/nl-covid19-data-dashboard/compare/epic/COR-1475-safety-region-removal...feature/COR-1498-vaccinations-toggle-button-removal?expand=1#diff-682a0b5666a3b5a4d56ff6f94bf6174f38abd387c77eff6d36177fb6ab1372af)
- Merged together `vaccine-coverage-choropleth_vr_and_gm.tsx` to `vaccine-coverage-choropleth.tsx `
- Removed `vaccine-coverage-choropleth_vr_and_gm.tsx`

## Screenshots
**_Before_**
<details>

  <summary>Toggle before screenshot</summary> 

Screenshot 2023-03-29 at 13 55 27

</details>

**_After_**
<details>
  <summary>Toggle after screenshot</summary> 

Screenshot 2023-03-29 at 13 55 04

</details>

### CODE REVIEW

**Review by VWSCoronaDashboard26 — DISMISSED** [review]

**Review by VWSCoronaDashboard18 — CHANGES_REQUESTED** [review]
See the comments on the CSV file. (The rest is awesome!)

**Review by VWSCoronaDashboard26 — APPROVED** [review]

File: packages/cms/src/lokalize/key-mutations.csv [hunk 1]
```diff
@@ -471,5 +471,9 @@ timestamp,action,key,document_id,move_to
 2023-03-27T13:01:45.445Z,delete,common.accessibility.charts.elderly_at_home_deceased_over_time_chart.label,G1DXw0RdifOml06twMjRl8,__
 2023-03-28T13:00:33.765Z,delete,common.accessibility.charts.sewer_region_choropleth.description,cl0A2qFve1QxscrDIqKL1F,__
 2023-03-28T13:00:33.766Z,delete,common.accessibility.charts.sewer_region_choropleth.label,Z3v1SKYekK4CLDpzUPc5x5,__
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard27) [reply packages/cms/src/lokalize/key-mutations.csv]: Done

File: packages/cms/src/lokalize/key-mutations.csv [hunk 2]
```diff
@@ -471,5 +471,9 @@ timestamp,action,key,document_id,move_to
...
 2023-04-03T15:45:47.784Z,delete,pages.behavior_page.nl.tooltip_labels.compliance,uFGMarfssEPhdNlk3s6Avk,__
 2023-04-03T15:45:47.785Z,delete,pages.behavior_page.nl.tooltip_labels.support,EaUS1FZKh46VBcUuh47mI4,__
+=======
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard27) [reply packages/cms/src/lokalize/key-mutations.csv]: Done

File: packages/cms/src/lokalize/key-mutations.csv [hunk 3]
```diff
@@ -471,5 +471,9 @@ timestamp,action,key,document_id,move_to
...
+2023-03-29T10:43:54.266Z,delete,common.choropleth.choropleth_vaccination_coverage.shared.vr,G6M7GhzH8XbtdKiRrez14Y,__
+>>>>>>> 433f97e7e (fix: merged conflicts)
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard27) [reply packages/cms/src/lokalize/key-mutations.csv]: Done

File: packages/app/src/components/choropleth/logic/thresholds.ts [hunk 4]
```diff
@@ -335,7 +335,6 @@ export const thresholds: Thresholds = {
     travel: situationsThreshold,
     hospitality: situationsThreshold,
     fully_vaccinated_percentage: vaccineCoveragePercentageThresholds,
-    primary_series_percentage: vaccineCoveragePercentageThresholds,
     autumn_2022_vaccinated_percentage: vaccineCoveragePercentageThresholds,
```
Comment (VWSCoronaDashboard21) [inline packages/app/src/components/choropleth/logic/thresholds.ts:338]: Maybe out of scope, but could we also remove these thresholds?

**Review by VWSCoronaDashboard21 — APPROVED** [review]
Looks good to me!

**Pull request merged**